commands/command_track.go

```diff
@@ -17,6 +17,10 @@ import (
 )
 
 var (
+	blacklistedPrefixes = []string{
+		".git", ".lfs", // TODO
```

Comment:
I don't think so. I left it in to see if there were any other prefixes that we wanted to block out, but I think this list is fine for now 👍